odoo/fields.py

```diff
@@ -3487,7 +3560,7 @@ def _add_missing_names(cls, values_list):
                 definition['name'] = str(uuid.uuid4()).replace('-', '')[:16]
 
     @classmethod
-    def _parse_json_types(cls, values_list, env, check_existence=True):
+    def _parse_json_types(cls, values_list, env):
```

Comment:
Does this rule really apply here ? (it's not on a model, so the probability that someone already overwrite / use it is extremely low, and it would be better to remove useless option to might bring confusion...)

Reply:
A bit of both ... well ok I guess, anyway framework was ok with the code :) . Let us be less catholic than the pope (hem).